ExampleMod/ExamplePlayer.cs

```diff
@@ -650,25 +650,25 @@ public class ExamplePlayer : ModPlayer
 			Player drawPlayer = drawInfo.drawPlayer;
 			Mod mod = ModLoader.GetMod("ExampleMod");
 
-
-			// To figure out when we should draw our ExampleBreastplate glowmask, we make use of GetEquipSlot.
-			// It takes in the name of the item that comes with an equip texture, and then its EquipType.
+			// To figure out when we should draw our ExampleBreastplate glowmask, we make use of GetEquipSlot
+			// It takes in the name of the item that comes with an equip texture, and then its EquipType
 			// By comparing that to drawPlayer.body, which represents the currently visible body armor, we make sure it's safe to draw ours
 			// Here we abort if the currently equipped body armor isn't what we need for our glowmask
 			if (drawPlayer.body != mod.GetEquipSlot("ExampleBreastplate", EquipType.Body)) {
 				return;
 			}
 
+			// The texture we want to display on our player
+			Texture2D texture;
 			// Because we have a breastplate glowmask, and breastplates have different textures depending on if the player is male or female, we have to use two textures here
-			string gender = "";
-			if (!drawPlayer.Male) {
-				gender = "Female";
+			if (drawPlayer.Male) {
```

Comment:
Still we really prefer the elvis operator for assignments: `texture = drawplayerMale ? maleoption : femaleoption`

Reply:
I was under the impression that ternary operator is not very newbie friendly since this is a very basic example/walkthrough. I very rarely see them in Examplemod too.

Reply:
Plus with the length of the single line you would have to line break after the "?" which is always confusing for people new to code (the fact that you can break the same line/line without a ";" at the end)

Reply:
I think using the ternary operator is fine here because Player Layers are already decently advanced topics, I don't think newbies should be attempting this.